wire-compiler/api/wire-compiler.api

```diff
@@ -196,21 +196,6 @@ public final class com/squareup/wire/schema/TargetKt {
 public final class com/squareup/wire/schema/WireRun {
 	public fun <init> (Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/util/List;Ljava/util/Map;ZLjava/util/List;)V
 	public synthetic fun <init> (Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/util/List;Ljava/util/Map;ZLjava/util/List;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
-	public final fun component1 ()Ljava/util/List;
```

Comment:
Are we fine with breaking compatibility?

Reply:
yes! Anybody using WireRun as a data class doesn't deserve attention!